modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -23,15 +23,15 @@ def request_representative
                   @claims_api_forms_validation_errors
           end
 
-          bgs_form_attributes = form_attributes.deep_merge(veteran_data)
-          bgs_form_attributes.deep_merge!(claimant_data) if user_profile&.status == :ok
-          bgs_form_attributes.deep_merge!(representative_data)
-          bgs_form_attributes.deep_merge!(organization_data) if @organization
+          bgs_form_attributes = build_bgs_attributes(form_attributes)
 
-          ClaimsApi::PowerOfAttorneyRequestService::Orchestrator.new(target_veteran.participant_id,
-                                                                     bgs_form_attributes.deep_symbolize_keys,
-                                                                     user_profile&.profile&.participant_id,
-                                                                     :poa).submit_request
```

Comment:
@jvcAdHoc Recognizing that this was here already, and what do you think about adding the safe navigator here?

```target_veteran&.participant_id,
                                                                     bgs_form_attributes&.deep_symbolize_keys,
                                                                     user_profile&.profile&.participant_id,
                                                                     :poa)&.submit_request```

Reply:
This creates a class instance
```ruby
ClaimsApi::PowerOfAttorneyRequestService::Orchestrator.new(target_veteran.participant_id, bgs_form_attributes.deep_symbolize_keys, user_profile&.profile&.participant_id, :poa)
```

And then the `submit_request` method is called on that instance. It would be unusual for the instantiation to fail in a way that it returned `nil`

Reply:
And we'd probably want the nil class error raised when calliling `submit_request` in that scenario becuase otherwise it would look like a successful request when nothing really happened in CorpDB/BGS